src/model/wallet.ts

```diff
@@ -205,6 +211,14 @@ const authenticationPrompt = lang.t('wallet.authenticate.please');
 
 export const createdWithBiometricError = 'createdWithBiometricError';
 
+const isHardwareWalletKey = (key: string | null) => {
+  const data = key?.split('/');
+  if (data && data.length > 1) {
+    return true;
+  }
+  return false;
+};
```

Comment:
Nit: `key` could be more clear as `privateKey` or `pkey`.

Maybe dumb question for my own purposes (and maybe a comment if you think it'd be helpful): what's the slash in a hardware pkey for? Is that a standard?

Reply:
we're saving the deviceID/index as the pkey since thats what we need to connect and request the correct info from the device